common/src/main/java/com/redhat/cloud/notifications/models/Endpoint.java

```diff
@@ -49,6 +49,8 @@
 
     private Boolean enabled = Boolean.FALSE;
 
+    private EndpointStatus status = EndpointStatus.UNKNOWN;
```

Comment:
Does Hibernate ORM automatically converts the enum members into their ordinal? We use `EndpointTypeConverter` for the type but maybe we don't need it.

Reply:
Yes it does the ordinal conversion by default.